### PR TITLE
Client side fix

### PR DESCRIPTION
**Pull request by timothycobrien**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -13,8 +13,8 @@ services:
       - "FLASK_ENV=development"
       - "DATABASE_URI=postgresql://postgres:postgres@postgres:5432/postgres"
       - "DATABASE_LOG=True"
-    ports:
-      - "5000:5000"
+    # ports:
```
Comment (EvanW1999) [inline docker-compose.yml:16]: Why adding this back?

Comment (timothycobrien) [reply docker-compose.yml:16]: Isn't this commenting out the ports

Comment (EvanW1999) [reply docker-compose.yml:16]: Oh yeah. But aren't they already commented out on origin/main

Comment (timothycobrien) [reply docker-compose.yml:16]: Yea they are, which is why this is weird

Comment (EvanW1999) [reply docker-compose.yml:16]: Branch is 4 commits ahead 1 commit behind, so just pull in and rebase.

Comment (timothycobrien) [reply docker-compose.yml:16]: Yea I'll rebase on top of the other PR, which has more things I think anyway, and close this one

**Pull request closed without merging**